code/modules/reagents/chemistry/machinery/chem_dispenser.dm

```diff
@@ -87,11 +87,11 @@
 
 /obj/machinery/chem_dispenser/Initialize()
 	. = ..()
-	dispensable_reagents = sortList(dispensable_reagents, /proc/cmp_reagents_asc)
+	dispensable_reagents = dispensable_reagents
```

Comment:
The subtypes of this chem dispensers do not have sorted predefined lists.
This will results in the order of chems of, for example, the booze and soda dispenser to have a different order.
Not saying you need to change that, just letting you know.

Reply:
Will revert changes here since I believe it could be better. Thanks!